common/io/Serial.cpp

```diff
@@ -282,20 +282,26 @@ void ReleaseUUCPLock(const std::string &path) {
 
   pid_t our_pid = getpid();
   if (our_pid == locked_pid) {
-    if (RemoveLockFile(lock_file)) {
+    if (RemoveUUCPLockFile(lock_file)) {
       OLA_INFO << "Released " << lock_file;
     }
   }
-#endif  /* else no action needed */
 }
 
-bool AcquireLockAndOpen(const std::string &path, int oflag, int *fd) {
+bool AcquireLockAndOpenSerialPort(const std::string &path, int oflag, int *fd) {
```

Comment:
How come this only works for a serial port? Aside from the fact we've got it in Serial.cpp? Won't it work in any file?

Reply:
Technically, flock() will work for any file of course.  UUCP wouldn't, though, because the lockfile name is made from the base name.  Semantically, the UUCP method is not really defined for anything other than a serial port.  So I don't think it makes sense to expand these routines to other things.